docs/README_human_mesh.md

```diff
@@ -0,0 +1,294 @@
+# Human Mesh Estimation
```

Comment:
The filename does not look right.
Where did we put those files before, `data_preparation.md`?

Reply:
Since this is a different task (human mesh recovery), maybe we should use a separate README file?
Need further discussion.

Reply:
I see, this is a doc for tasks. Previously supported tasks haven't got their readmes yet.
How about put this file at `configs/mesh/README.md`?
Later we can add `configs/hand/README.md`, etc.

Reply:
We can use script to gather them into the docs site

Reply:
agree